### PR TITLE
feat: unstoppable domain integration 

### PR DESCRIPTION
**Pull request by amritkumarj**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,7 @@
     "@uniswap/v2-core": "^1.0.1",
     "@uniswap/v2-periphery": "^1.1.0-beta.0",
     "@uniswap/v2-sdk": "^3.0.1",
+    "@unstoppabledomains/resolution": "^8.3.1",
```
Comment (kraikov) [inline package.json]: can you please pin the version

**Review by kraikov — CHANGES_REQUESTED** [review]

File: src/utils/uns.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                return 'RSK'
+            case ChainId.Ethereum:
+            case ChainId.Arbitrum:
+            default:
+                return 'ETH'
+        }
+
+    }
+
+    async lookupDomain(address: string, chainId: ChainId): Promise<string | null> {
```
Comment (kraikov) [inline src/utils/uns.ts]: Can you use `Nullable<string>` instead of `string | null`?

File: src/utils/uns.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
+import { HttpClient } from '@chainify/client';
```
Comment (monokh) [inline src/utils/uns.ts]: `utils` seems the wrong place for this
how about `src/nameResolvers/uns.ts` ?

File: src/build.config.ts [hunk 4]
```diff
@@ -208,6 +211,9 @@ const config: WalletCoreConfig = {
     ChainId.Avalanche,
   ],
   supportedBridgeAssets: ['MATIC', 'RBTC', 'AVAX'],
+  resolutionService : 'https://unstoppabledomains.g.alchemy.com/domains/',
```
Comment (monokh) [inline src/build.config.ts]: This should be grouped in some way
perhaps?
```
nameResolvers: {
  uns: {
    resolutionService: ...,
    tldAPI: ...,
    alchemyKey: ...,
  }
}
```

**Review by monokh — CHANGES_REQUESTED** [review]
Looking good!
just some small comment

**Review by monokh — APPROVED** [review]

**Pull request merged**